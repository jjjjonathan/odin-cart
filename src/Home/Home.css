h2 {
  text-align: center;
  font-size: 5rem;
}

.lead {
  font-size: 2rem;
  font-weight: 300;
  text-align: center;
}
