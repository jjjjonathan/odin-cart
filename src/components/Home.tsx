const Home = () => (
  <div>
    <h2>Welcome to Odincart.</h2>
    <p className="lead">Where you can't buy anything, but you can pretend.</p>
  </div>
);

export default Home;
