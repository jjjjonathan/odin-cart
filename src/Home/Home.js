import React from "react";
import "./Home.css";

export default function Home(props) {
  return (
    <div>
      <h2>Welcome to Odincart.</h2>
      <p className="lead">Where you can't buy anything, but you can pretend.</p>
    </div>
  );
}
