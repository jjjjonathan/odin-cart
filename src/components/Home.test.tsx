import Home from './Home';
import renderer from 'react-test-renderer';

describe('Home Component', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
