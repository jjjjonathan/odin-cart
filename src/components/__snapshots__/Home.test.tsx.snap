// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component should match snapshot 1`] = `
<div>
  <h2>
    Welcome to Odincart.
  </h2>
  <p
    className="lead"
  >
    Where you can't buy anything, but you can pretend.
  </p>
</div>
`;
